config/database.yml

```diff
@@ -17,6 +17,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: postgres
   # For details on connection pooling, see Rails configuration guide
```

Comment:
This `username: postgres` addition makes it impossible for me to run your migrations. Consider reverting this change, please. Otherwise, you will have to include more instructions about setting up Postgres for your project.